apps/mosaico/static/fragor-och-svar.html

```diff
@@ -0,0 +1,460 @@
+<div class="static-page__faq">
+  <h1 class="faq__headline">Frågor och svar</h1>
+  <div class="faq__qa-container">
+    <div class="faq__qa-section faq-top">
+      <h2 class="faq__section-headline">Vanliga frågor</h2>
+      <div class="faq__qa qa__top">
+        <button class="faq__question">Vad är Premium? Varför måste jag ha en
+          prenumeration att läsa en del av artiklarna?</button>
+        <p class="faq__answer faq__answer--open answer">En del av vårt material är endast tillgängligt för våra
+          prenumeranter (som
+          gör det
+          möjligt för oss att existera). Vi markerar sådana artiklar med markering PREMIUM. Du kan
+          teckna en prenumeration för att få tillgång till Premiummaterialet. Gör din beställning på
+          <a href="https://prenumerera.ksfmedia.fi/">prenumerera.ksfmedia.fi</a>.
+        </p>
+      </div>
+      <div class="faq__qa qa__top">
+        <button class="faq__question">Jag är prenumerant, men kommer ändå inte åt
+          Premiuminnehållet. Vad ska jag göra?</button>
+        <p class="faq__answer faq__answer--open answer">Logga in med din e-postadress som är kopplad till
```

Comment:
Not sure if the `answer` class is being styled in the less file? I couldn't find it there.

Reply:
No it's not atm, it was for testing purpose